.ko.yaml

```diff
@@ -0,0 +1,9 @@
+---
+builds:
+  - id: imgctrl
+    main: ./cmd/imgctrl
+    flags:
+      - -tags
+      - containers_image_openpgp
+    ldflags:
+      - -X main.Version=v0.0.0
```

Comment:
@imjasonh, @ricardomaraschini, do we have a Golang environment variable (go env) to set this flag? So we could do in a single place (`Makefile`).